operations/research-and-development/product/development-process/user-interface-text-guidelines.md

```diff
@@ -84,47 +93,113 @@ An error message informs the user that something went wrong. Errors prevent the
 
-## Navigation labels
+### Guidelines for punctuation
 
-Navigation labels should be as short as possible and support the user in finding their way.
+**Headings**
 
-### Input labels
+These are H1 headings such as the title of a modal. Titles shouldn't have periods unless the headline is more than one sentence. Titles also shouldn't contain punctuation such as question marks, colons, semi-colons.
 
-Input labels should be as short and concise as possible and describe the input field.
+**Paragraphs**
 
-### Capitalization
+Paragraphs should always be properly punctuated.
 
-**Titles**
+**Bullets**
 
-Titles should use sentence case \(e.g. **This article is about user interface content**\) with the exception of proper nouns \(e.g. **This article is about Mattermost**\).
+Bullet lists shouldn't have periods unless the bullet text is more than one sentence.
 
-**Button labels, input labels, and navigation labels**
+**Button labels**
 
-These labels should use sentence case. \(e.g. **This article is about user interface content**\) with the exception of proper nouns \(e.g. **This article is about Mattermost**\).
+Button labels shouldn't have periods or other punctuation.
 
-### Punctuation
+### Guidelines for capitalization
 
-**Titles**
+We follow the same capitalization rules across all our documentation assets, including in-product text and UI elements: Always use sentence case (except where you're using a proper noun, of course). This applies to:
 
-Titles shouldn't have periods unless the headline is more than one sentence.
+* Page titles
+* Page headings
+* Section headings
+* Button labels
+* Input labels
+* Navigation labels
+* Menu items
 
-**Paragraphs**
+**Instead of:** This Article is About Mattermost
+**Use:** This article is about Mattermost
 
-Paragraphs should always be properly punctuated.
+### How we write things (WIP)
 
-**Bullets**
+As previously mentioned, we write with empathy in the context of achievement, and in natural language. This can be quite hard to do; as technical writers we generally lean toward very clear and concise writing that can feel clinical. Writing more naturally means we try to avoid convoluted phrasing and we try to make things more simple. 
 
-Bullets lists shouldn't have periods unless the bullet text is more than one sentence.
+So, instead of saying: "When testing the Java app ensure your third-party connection is secure before initiating the test sequence." rather say: "Make sure your third-party connection is secure before testing the app."
 
-**Button labels**
+Because we strive to use natural language, the way we phrase microcopy isn't templated - that would defeat the point. Instead, here are some examples of microcopy we use for various features and products. The common theme is clarity and empathy.
 
```

Comment:
I'll add some screenshots of the playbooks microcopy here.

Reply:
Great idea!